assets/stylesheets/components/_user-login.scss

```diff
@@ -0,0 +1,10 @@
+.user-login {
```

Comment:
Since both components `user-login` and `user-registration` seem identical, how about keeping only one `user-auth` component?